directories/src/redux/epics/node-epics.js

```diff
@@ -96,9 +97,12 @@ const requestBrokerEpic = (action$, store) => {
       .mergeMap(({ txid, trytesArray }) =>
         Observable.fromPromise(
           brokerNode.completeBrokerNodeAddressPoW(txid, trytesArray[0])
-        ).map(({ data }) => {
+        ).flatMap(({ data }) => {
```

Comment:
Nit: Use `mergeMap` instead of `flatMap` to be consistent with the rest of the codebase.